src/validation/forks.cpp

```diff
@@ -95,3 +95,21 @@ bool IsNov2018Activated(const Consensus::Params &consensusparams, const CBlockIn
     }
     return IsNov2018Activated(consensusparams, pindexTip->nHeight);
 }
+
+bool IsNov2019Enabled(const Consensus::Params &consensusparams, const CBlockIndex *pindexTip)
+{
```

Comment:
`pindexTip`? This looks to be parent of tip. So `prevTip`? `parentOfTip`?

Reply:
`IsNov2019Enabled` is wrapper around `IsforkActiveOnNextBlock` which return `true` if the fork will be enabled at the next block in respect to the current tip (see `src/chain.cpp`).

So if we use `c` as a label for the activation block, this function will return true for every block `x` with height that belongs to the interval `[h(c-1),+inf]`. (`h()` return the height of a block).

You could use that helper with any block, this is not meant to be used with current tip or the parent of the current tip. 

E.g. in `DisconnectTip()` we used twice, once with the block that is going to be disconnected and the with its parent. 

```c++
if (IsNov2019Enabled(consensusParams, pindexDelete) && 
       !IsNov2019Enabled(consensusParams, pindexDelete->pprev))
```

So that the condition above will return true only if we are disconnecting the block which activated the fork, i.e. we have a reorg across fork activation block.